WebGL.md

```diff
@@ -0,0 +1,65 @@
+# Implementing Unity-Immutable-SDK for WebGL
+
+This guide provides focused instructions for implementing the Unity-Immutable-SDK in WebGL projects.
+
+For general information on the SDK, please refer to
+the [Unity-Immutable-SDK documentation](https://docs.immutable.com/sdks/zkEVM/unity).
+
+Live example can be found at https://immutable.github.io/unity-immutable-sdk/sample/webgl
+
+## WebGL Template Setup
+
+1. Navigate to `sample/Assets/WebGLTemplates/unity-webview/`.
+
+2. Copy the following files to your WebGLTemplates folder:
+   - `unity-webview.js`
+   - `callback.html`
+   - `logout.html`
+   - `Passport/index.html`
+
+3. Add the following script tag to your `index.html`:
+   ```html
+   <script src="unity-webview.js"></script>
+   ```
+
+## PKCE Login and Logout Implementation
+WebGL supports only PKCE flow. Follow these steps for implementation:
+
+1. Define a deep link scheme for your game:
+   - Login: https://game.domain.com/callback.html
+   - Logout: https://game.domain.com/logout.html
```

Comment:
Games could still rename these files as `someCallbackFile.html` and `someLogoutFile.html` and use these in Hub instead, right? If so, it would be good to tell them that. They might think the redirect URIs need to be `/callback` and `/logout` only.

Reply:
good point will update.